plugins/github-actions/src/components/Cards/RecentWorkflowRunsCard.tsx

```diff
@@ -70,15 +72,19 @@ export const RecentWorkflowRunsCard = ({
       }
     />
   ) : (
-    <TableContainer component={Card}>
+    <InfoCard
+      title="Recent Workflow Runs"
+      subheader={branch ? `Branch: ${branch}` : 'All Branches'}
+      noPadding
+      variant={variant === 'grid-item' ? 'height100' : ''}
+    >
```

Comment:
Also changed the RecentWorkflowRunsCard to an InfoCard to apply the `grid-item` variant

![Bildschirmfoto 2020-10-09 um 16 49 40](https://user-images.githubusercontent.com/61695677/95598683-83cf6280-0a50-11eb-9589-0743005c151f.png)